### ISSUE TITLE
Call for Maintainers

### ISSUE THREAD

**Issue by paltman** [pinax/pinax-notifications#105]
Looking for maintainers!

There is a lot of pull requests and open issues that the current maintainers, myself included, are just not finding the time to properly get to.

Maybe you've submitted some PRs and are frustrated with the lack of attention. Maybe you use this project in one or more of your projects and want to see that it is properly carried forward.

Whatever you reasons may be, let me know if you have interest and I'll add you to the repo and to PyPI (will need your PyPI name).

Preference will go to those who have commits on this repo and/or have shown an active interest in the issues.

Thanks!
Patrick

**Comment by powderflask** [comment]
@paltman - I already maintain more packages than I ought to, but I'm willing to at least keep this one maintained.   At a minimum I can commit to processing the outstanding PR's and upgrading the package to run on Py3.11 and Dj4.2.  Can't promise to handle support or enhancement requests, but at least I could keep it alive.  Let me know.  My PyPi name is same.

**Comment by andharris** [comment]
@powderflask thanks for offering to help. I'm working on an app that is blocked on upgrading to Dj4.2 because of this library. Have you made any progress on updating to support Dj4.2? Please let me know if there is anything I could do to support updates needed to for Dj4.2.

**Comment by powderflask** [comment]
Yes.  I have a PR ready to go.  Using it in production, all working as expected.
You can get it here:
https://github.com/powderflask/pinax-notifications/tree/django4.2
I see several other attempts in the PR queue, so I didn't want to confuse things by adding another PR to the queue until someone is processing them.

**Comment by andharris** [comment]
@paltman any way we might be able to all @powderflask to start processing the outstanding PRs?

**Comment by pataquets** [comment]
@paltman : you might want to take a look at https://adoptoposs.org/
